scenarioo-server/src/main/java/org/scenarioo/rest/step/ScreenshotResource.java

```diff
@@ -50,11 +50,28 @@
 	 * This method is used internally for loading the image of a step. It is the faster method, because it already knows
 	 * the filename of the image.
 	 */
-	@GetMapping(path = "{scenarioName}/image/{imageFileName}", produces = "image/*")
-	public ResponseEntity getScreenshot(@PathVariable("branchName") final String branchName,
-			@PathVariable("buildName") final String buildName, @PathVariable("usecaseName") final String usecaseName,
-			@PathVariable("scenarioName") final String scenarioName, @PathVariable("imageFileName") final String imageFileName) {
+	@GetMapping(path = "{scenarioName}/image/{imageFileName}.png", produces = "image/png")
+	public ResponseEntity getScreenshotPng(@PathVariable("branchName") final String branchName,
+										   @PathVariable("buildName") final String buildName, @PathVariable("usecaseName") final String usecaseName,
+										   @PathVariable("scenarioName") final String scenarioName, @PathVariable("imageFileName") final String imageFileName) {
 
+		return getScreenshot(branchName, buildName, usecaseName, scenarioName, imageFileName + ".png");
+	}
+
+	/**
+	 * This method is used internally for loading the image of a step. It is the faster method, because it already knows
+	 * the filename of the image.
+	 * Additional method with produces=image/jpeg is needed, as IE 11 is a bit picky in its accept headers, so produces image/* does not work for jpeg images.
```

Comment:
should we provide the comment also on the other method?
And: is it IE that is picky or Firefox? Because on the other endpoint we talk about Firefox. that is picky with headers.